src/stateProviders/provideSavedQueries.js

```diff
@@ -26,7 +27,9 @@ export default provideState({
         )
         .then(value => state => {
           effects.setLoading(false);
-          return { ...state, queries: value || [] };
+          const queries = value.filter(q => !q.content.example) || [];
+          const exampleQueries = value.filter(q => q.content.example) || [];
```

Comment:
`(value || []).filter` is a better fallback because if `value` is falsey or not an array, `.filter(...)` would still cause an error